kratos/tests/cpp_tests/external_libraries/test_delaunator.cpp

```diff
@@ -0,0 +1,194 @@
+    validate(coordinates, 99745);
+//     validate(coordinates, 99745, "test1");
+}
+
+/**
+ * Checks that Delaunator works for issue 11 of delaunator-cpp
+ */
+KRATOS_TEST_CASE_IN_SUITE(DelaunatorIssues11, KratosExternalLibrariesFastSuite)
+{
+    std::vector<double> coordinates = { 516, 661, 369, 793, 426, 539, 273, 525, 204, 694, 747, 750, 454, 390 };
+    validate(coordinates, 244863);
+//     validate(coordinates, 244863, "test2");
+}
+
+/**
+ * Checks that Delaunator works for issue 13 of delaunator-cpp
+ */
+KRATOS_TEST_CASE_IN_SUITE(DelaunatorIssues13, KratosExternalLibrariesFastSuite)
+{
+    std::vector<double> coordinates = { 4, 1, 3.7974166882130675, 2.0837249985614585, 3.2170267516619773, 3.0210869309396715, 2.337215067329615, 3.685489874065187, 1.276805078389906, 3.9872025288851036, 0.17901102978375127, 3.885476929518457, -0.8079039091377689, 3.3940516818407187, -1.550651407188842, 2.5792964886320684, -1.9489192990517052, 1.5512485534497125, -1.9489192990517057, 0.44875144655029087, -1.5506514071888438, -0.5792964886320653, -0.8079039091377715, -1.394051681840717, 0.17901102978374794, -1.8854769295184561, 1.276805078389902, -1.987202528885104, 2.337215067329611, -1.6854898740651891, 3.217026751661974, -1.021086930939675, 3.7974166882130653, -0.08372499856146409 };
```

Comment:
Here you could also use std::array, might be slightly more efficient

Reply:
Okay

Reply:
The problem with std::array is that implies to define it as template, and I don't like that...

Reply:
you mean the size?
I thought there was a way that the std::array determines this from the intializer list
anyway, not important

Reply:
https://en.cppreference.com/w/cpp/container/array This requires not only the type, but the size which is fixed, so it can be done, but templating the method and moving all triangle stuff to the header